uilayer.cpp

```diff
@@ -279,12 +279,8 @@ ui_layer::update() {
 
         case( GLFW_KEY_F1 ) : {
             // f1, default mode: current time and timetable excerpt
-            auto const &time = simulation::Time.data();
             uitextline1 =
-                "Time: "
-                + to_string( time.wHour ) + ":"
-                + ( time.wMinute < 10 ? "0" : "" ) + to_string( time.wMinute ) + ":"
-                + ( time.wSecond < 10 ? "0" : "" ) + to_string( time.wSecond );
+                "Time: " + std::string( simulation::Time );
```

Comment:
Samo użycie operatora + nie wymusi konwersji do std::string?

Reply:
Gdybym wiedział, to bym tak zrobił ;D

Reply:
niee, lepiej jawnie.

Reply:
Jedyne co widzę, to to, że zamiast jawnej konwersji, tworzę tu nowego tempa. Tzn zamiast:
`(std::string)(simulation::Time)` mam `std::string( simulation::Time )`, tzn tworzę stringa ze stringa. Poprawić?

Reply:
Dobrze rozumiem, że samo użycie ```(...) + simulation::Time``` tworzy stringa? Jeżeli tak, to aktualne rozwiązanie tworzy stringa ze stringa, a rzutowanie zamienia stringa na stringa.